test/bench/helpers.hpp

```diff
@@ -0,0 +1,77 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2019 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include "test/utils/utils.hpp"
+#include <benchmark/benchmark.h>
+#include <evmc/evmc.hpp>
+#include <evmone/analysis.hpp>
+
+namespace evmone::test
+{
+extern std::map<std::string_view, evmc::VM> registered_vms;
+
+constexpr auto gas_limit = std::numeric_limits<int64_t>::max();
+
+inline void analyse(benchmark::State& state, bytes_view code) noexcept
+{
+    auto bytes_analysed = uint64_t{0};
+    for (auto _ : state)
+    {
+        auto r = evmone::analyze(EVMC_ISTANBUL, code.data(), code.size());
+        benchmark::DoNotOptimize(r);
+        bytes_analysed += code.size();
+    }
+
+    using benchmark::Counter;
+    state.counters["size"] = Counter(static_cast<double>(code.size()));
+    state.counters["rate"] = Counter(static_cast<double>(bytes_analysed), Counter::kIsRate);
+}
+
+inline evmc::result execute(evmc::VM& vm, bytes_view code, bytes_view input) noexcept
+{
+    auto msg = evmc_message{};
+    msg.gas = gas_limit;
+    msg.input_data = input.data();
+    msg.input_size = input.size();
+    return vm.execute(EVMC_ISTANBUL, msg, code.data(), code.size());
```

Comment:
Since `EVMC_ISTANBUL` is duplicated twice in this file, perhaps a `constexpr evmc_version` on the top might be a good idea to have a single place for the setting? We don't have `EVMC_LATEST`, right? I think we removed that.